OrbitGl/App.cpp

```diff
@@ -805,15 +805,27 @@ void OrbitApp::LoadModules(int32_t process_id,
 //-----------------------------------------------------------------------------
 void OrbitApp::LoadModulesFromPreset(const std::shared_ptr<Process>& process,
                                      const std::shared_ptr<Preset>& preset) {
-  std::vector<std::shared_ptr<Module>> modules;
+  std::vector<std::shared_ptr<Module>> modules_to_load;
   for (const auto& pair : preset->m_Modules) {
     const std::string& module_path = pair.first;
     const auto& module = process->GetModuleFromPath(module_path);
-    if (module != nullptr && !module->IsLoaded()) {
-      modules.emplace_back(std::move(module));
+    if (module == nullptr) {
+      SendErrorToUi(
```

Comment:
This does potentially spam the user with dialog boxes, does it? Maybe we should have one error dialog at the end?

Reply:
I would judge this as a case that should rarely happen, but your idea is good nevertheless. I aggregated the modules that are not found and send the error after the loop